scripts/travis-test.sh

```diff
@@ -18,17 +18,19 @@
 set -o nounset
 set -o errexit
 
+RTN=0
 SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd -P)"
 source "${SCRIPT_DIR}/travis-common.sh"
 
 log_and_run() {
   echo $*
-  exec $*
+  (exec $*) && true
```

Comment:
Probably a silly question, but what is the `&& true` supposed to do here? I'm no Travis expert, but in a Bourne shell that's just a nop, right?